F1WM.UnitTests/Controllers/RSSControllerTests.cs

```diff
@@ -60,7 +60,7 @@ public async Task ShouldReturnCachedRSS()
 
 			serviceMock.Verify(s => s.GetFeed(null), Times.Never);
 			cachingServiceMock.Verify(c => c.Get<SyndicationFeed>(It.IsAny<string>()), Times.Once);
-			cachingServiceMock.Verify(c => c.Set(It.IsAny<string>(), It.IsAny<SyndicationFeed>(), It.IsAny<MemoryCacheEntryOptions>()), Times.Never);
+			cachingServiceMock.Verify(c => c.Set(It.IsAny<string>(), It.IsAny<SyndicationFeed>(), TimeSpan.Zero), Times.Never);
```

Comment:
We should make sure that `Set` isn't called with *any* param, so `TimeSpan.Zero` should be changed to fitting `It.IsAny` setup. The same applies to all new assertions that check for `Times.Never` and have specific `TimeSpan` params